data/data/bootstrap/baremetal/files/usr/local/bin/startironic.sh.template

```diff
@@ -87,7 +86,6 @@ podman wait -i 1000 ipa-downloader
 podman wait -i 1000 coreos-downloader
 while ! curl --fail http://localhost/images/rhcos-ootpa-latest.qcow2.md5sum ; do sleep 1; done
 while ! curl --fail --head http://localhost/images/ironic-python-agent.initramfs ; do sleep 1; done
-while ! curl --fail --head http://localhost/images/ironic-python-agent.tar.headers ; do sleep 1; done
```

Comment:
The [rhel based IPA image](https://github.com/openshift/ironic-ipa-downloader/blob/master/get-resource.sh#L16) doesn't produce this file, so the move to openshift images made this wait forever.